auraed/src/runtime/cell_service/executables/auraed/nested_auraed.rs

```diff
@@ -215,11 +215,14 @@ impl NestedAuraed {
         //       (contrary to ^^^). What do we want to signal?
 
         if wants_isolated_pid(&self.shared_namespaces) {
-            // TODO: Here, SIGINT works when using auraescript, but hangs(?) during unit tests.
-            //       SIGKILL, however, works in both scenarios. Why?
-            self.process.kill(Some(SIGKILL))?;
+            // TODO: Here, SIGTERM works when using auraescript, but hangs(?) during unit tests.
+            //       SIGKILL, however, works in both scenarios. This may be solved once we proxy
+            //       signals to nested auraeds.
+            self.process.kill(Some(SIGTERM))?;
             self.process.wait()
         } else {
+            // TODO: https://aurae.io/signals/ seems to indicate that we always want
+            //       to send SIGTERM to nested auraeds?
             self.process.kill(SIGKILL)?;
```

Comment:
i think SIGTERM too, but we should also handle that signal and shut down as cleanly as possible.